### ISSUE TITLE
CSS Styling Issues

### ISSUE THREAD

**Issue by wresendiz1** [wresendiz1/personal-web#1]
Fix hover icons on dark theme, change fonts, ensure spacing is approriate

**Comment by callmeVineeth** [comment]
Hi I would like to work on this, assign this task to me

**Comment by EditsbyVineeth** [comment]
Yes I noticed that some icons are not hovering while dark theme, and buttons like Home, Coursework and Projects  fontcolor is not changing while hovering in dark them. I would like to work on this issue, Pls assign this task to me
